Fix hoauth2 compat for 2.7.0 (#165)
freckle/yesod-auth-oauth2: CHANGELOG.md

## [_Unreleased_](https://github.com/thoughtbot/yesod-auth-oauth2/compare/v0.7.0.3...main)

## [v0.7.0.3](https://github.com/thoughtbot/yesod-auth-oauth2/compare/v0.7.0.2...v0.7.0.3)

> - Support `hoauth-2.7`, this change is breaking if something other than
>   than `fetchAccessToken` or `fetchAccessToken2` is used build the plugin (which
>   we don't believe to be the case)

Reviewer: ```suggestion
- Support `hoauth-2.7`. This change is only breaking in the unlikely case of users
  using something other than `fetchAccessToken` or `fetchAccessToken2`
```

> ## [_Unreleased_](https://github.com/thoughtbot/yesod-auth-oauth2/compare/v0.7.0.3...main)

Reviewer: I'd either remove this (and version bump in its own PR), or include the `package.yaml` / `.cabal` change to `version` here too.
